task_netcore/task2/task2/Services/DataService.cs

```diff
@@ -0,0 +1,66 @@
+﻿using AutoMapper;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System.Net;
+using System.Threading.Tasks;
+using task2.Interfaces;
+using task2.Models;
+
+namespace task2.Services
+{
+    public class DataService : IDataService
+    {
+        private string url;
+        private readonly IMapper _mapper;
+        public DataService(IMapper mapper, IConfiguration configuration)
+        {
+            _mapper = mapper;
+            url = configuration["url"];
+        }
+
+        public ResponseModel GetInfo()
+        {
+            ForeignModel response = new ForeignModel();
+
+            using (WebClient client = new WebClient())
+            {
+                var content = client.DownloadString(url);
+                response = JsonConvert.DeserializeObject<ForeignModel>(content);
+
+                Counter(response);
```

Comment:
Could you change this name to something more obvious? Also, this method could be called after mapping for response model